CONTRIBUTING.md

```diff
@@ -29,73 +27,70 @@ If you are contributing for the first time - we would recommend reading
 > git branch --set-upstream-to=upstream/master master
 > ```
 >
-> After running these commands you'll be able to easily pull changes from the original repository by
+> After running these commands you'll be able to easily pull changes from the original repository with
 > `git pull`.
 
 ## Development
 
-1. Implement the hook in `src` folder.
-   - File should be named after the hook and placed in subdirectory also named after the hook.
-   - Hook should have return types explicitly defined.
-   - Hook should have JSDoc containing hook description and an overview of its arguments.
-   - Hook should be exported by name, not default exported.
-   - In case hook has some custom types as arguments or return values - it should also be exported.
-   - Types and interfaces should not have prefixes like `I` or `T` (it is only left in existing code
-     for compatibility reasons and will be removed).
-   - Hook should be developed with SSR in mind.
-   - In case hook is stateful and exposes `setState` method, or is has async callbacks (that can
+1. Implement the hook in the `src` folder.
+   - The file should be named after the hook and placed in a subdirectory also named after the hook.
+   - The hook should have return types explicitly defined.
+   - The hook should have a JSDoc comment containing a description of the hook and an overview of its arguments.
+   - The hook should be exported by name, not default exported.
+   - If the hook has custom types in its parameters or return values, they should be exported as well.
+   - Types and interfaces should not have prefixes like `I` or `T` (currently some types like this
+     still exists for compatibility reasons, but they will be removed in the future).
+   - The hook should be developed with SSR in mind.
+   - If the hook is stateful and exposes `setState` method, or is has async callbacks (that can
      theoretically be resolved after component unmount), it should use `useSafeState` instead of
      `useState`.
-   - In case of hooks reuse, import them as `import { useSafeState } from '../useSafeState/useSafeState';` instead of
+   - If you reuse hooks from this library, import them as `import { useSafeState } from '../useSafeState/useSafeState';` instead of
```

Comment:
Maybe this can be phrased a bit better. Maybe:
`If your hook reuses another `@react-hookz/web` hook...`?